cloudnatix/assets/dashboards/cloudnatix_overview.json

```diff
@@ -0,0 +1 @@
imeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes.memory.usage{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"custom_links":[{"label":"Analyze workloads","link":"https://app.cloudnatix.com/app/workloads"}]},"layout":{"x":5,"y":0,"width":2,"height":2}},{"id":5922853403113722,"definition":{"title":"Savings recommended","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.workload.monthly_projected_saving{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":false,"custom_unit":"$/month","custom_links":[{"label":"Discover insights","link":"https://app.cloudnatix.com/app/insights"}],"precision":2},"layout":{"x":7,"y":0,"width":3,"height":2}},{"id":6190142051270206,"definition":{"title":"instances","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:kubernetes_state.node.count{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":0,"y":2,"width":2,"height":1}},{"id":7170024749604466,"definition":{"title":"Workloads","title_size":"16","title_align":"left","type":"query_table","requests":[{"response_format":"scalar","formulas":[{"alias":"autopilot","formula":"query1","limit":{"count":500,"order":"desc"}},{"alias":"manual","formula":"query2"},{"alias":"not applied","formula":"query3"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:autopilot}","aggregator":"avg"},{"name":"query2","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:manual}","aggregator":"avg"},{"name":"query3","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:notapplied}","aggregator":"avg"}]}]},"layout":{"x":2,"y":2,"width":4,"height":2}},{"id":7654902037777398,"definition":{"title":"cores","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.cpu_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"custom_unit":"cores","precision":2},"layout":{"x":0,"y":3,"width":1,"height":1}},{"id":8825923597966732,"definition":{"title":"memory","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.memory_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":1,"y":3,"width":1,"height":1}}]},"layout":{"x":2,"y":0,"width":10,"height":5}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
```suggestion
{"title":"CloudNatix","description":"This dashboard depicts the overview of CloudNatix insights on cost reduction as well as the current spend and resource usage.","widgets":[{"id":485764349959394,"definition":{"type":"note","content":"![CloudNatix logo](https://docs.cloudnatix.com/_next/image?url=%2Fimgs%2Flogo.png&w=384&q=75)\n\nThis dashboard provides an overview on your cluster spending, as well as an overview of cost optimization insights that can be achieved using recommendations from CloudNatix.\n\nFor more details, please visit the following links:\n\n* [The detailed view of the workload improvements](https://app.cloudnatix.com/app/workloads)\n* [The details of CloudNatix insights](https://app.cloudnatix.com/app/insights)\n* [Cluster level information and recommendations](https://app.cloudnatix.com/app/cluster)\n* [CloudNatix documentation](https://docs.cloudnatix.com/)","background_color":"transparent","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":false},"layout":{"x":0,"y":0,"width":2,"height":5}},{"id":2615020295340226,"definition":{"title":"Overview","show_title":true,"type":"group","layout_type":"ordered","widgets":[{"id":3721013903265102,"definition":{"title":"Daily spend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.compute.daily_spend{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":false},"custom_links":[{"label":"Analyze workloads","link":"https://app.cloudnatix.com/app/workloads"}]},"layout":{"x":0,"y":0,"width":3,"height":2}},{"id":8596670878174568,"definition":{"title":"CPU used","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes.cpu.usage.total{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"custom_links":[{"label":"Analyze workloads","link":"https://app.cloudnatix.com/app/workloads"}]},"layout":{"x":3,"y":0,"width":2,"height":2}},{"id":2608502092823534,"definition":{"title":"Memory used","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes.memory.usage{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"custom_links":[{"label":"Analyze workloads","link":"https://app.cloudnatix.com/app/workloads"}]},"layout":{"x":5,"y":0,"width":2,"height":2}},{"id":5922853403113722,"definition":{"title":"Savings recommended","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.workload.monthly_projected_saving{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":false,"custom_unit":"$/month","custom_links":[{"label":"Discover insights","link":"https://app.cloudnatix.com/app/insights"}],"precision":2},"layout":{"x":7,"y":0,"width":3,"height":2}},{"id":6190142051270206,"definition":{"title":"instances","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:kubernetes_state.node.count{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":0,"y":2,"width":2,"height":1}},{"id":7170024749604466,"definition":{"title":"Workloads","title_size":"16","title_align":"left","type":"query_table","requests":[{"response_format":"scalar","formulas":[{"alias":"autopilot","formula":"query1","limit":{"count":500,"order":"desc"}},{"alias":"manual","formula":"query2"},{"alias":"not applied","formula":"query3"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:autopilot}","aggregator":"avg"},{"name":"query2","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:manual}","aggregator":"avg"},{"name":"query3","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:notapplied}","aggregator":"avg"}]}]},"layout":{"x":2,"y":2,"width":4,"height":2}},{"id":7654902037777398,"definition":{"title":"cores","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.cpu_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"custom_unit":"cores","precision":2},"layout":{"x":0,"y":3,"width":1,"height":1}},{"id":8825923597966732,"definition":{"title":"memory","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.memory_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":1,"y":3,"width":1,"height":1}}]},"layout":{"x":2,"y":0,"width":10,"height":5}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Reply:
A few grammatical improvements in the description/widget, but otherwise the dashboard looks good 👍